app/scripts/views/settings/avatar_change.js

```diff
@@ -45,15 +45,17 @@ define(function (require, exports, module) {
       }
     },
 
+    context: function () {
+      var account = this.getSignedInAccount();
```

Comment:
Testing this on 3 instances of firefox, I do see the issue if we have the same page already open on another browser, but if i refresh/signin/goto settings, this doesnt seem to show up. So I am guessing this is alright? 
@shane-tomlinson